_posts/0000-01-02-Harshita20ppt.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**Things may come to those who wait, but only the things left by those who hustle**